test/data/stages/cloud-init/b.mpp.yaml

```diff
@@ -31,6 +31,7 @@ pipelines:
         options:
           filename: 00-default_user.cfg
           config:
+            ssh_pwauth: false
```

Comment:
You'll need to regenerate the test manifests by running `sudo make test-data` in the repository.